### PR TITLE
Mukesh

### PR DESCRIPTION
**Pull request by mukeshkannanai**
#Task programe

### CODE REVIEW

**Comment by mukeshkannanai** [comment]
.

File: mukesh/AnimalsLeg.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+{
+
+    static int animals(int chick,int cow,int pig)
+    {
+        int ans=2*chick+4*(cow+pig);
+        return(ans);
+    }
+	public static void main(String[] args) {
+		Scanner sc=new Scanner (System.in);
+        System.out.println("Enter the count of animals(chick,cow,pig)");
```
Comment (pbravichandhar) [inline mukesh/AnimalsLeg.java]: indentation ?

File: mukesh/AnimalsLeg.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+		int chick=sc.nextInt();
+		int cow=sc.nextInt();
```
Comment (pbravichandhar) [inline mukesh/AnimalsLeg.java]: What happens when i give string

File: mukesh/ArrayDepth.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+             }
+
+         }
+        System.out.println(max);
+    }
+
+     public static void main(String []args){
+         Scanner sc=new Scanner (System.in);
+         System.out.println("Enter the Array");
+         String str=sc.next(); //Getting input as string
```
Comment (pbravichandhar) [inline mukesh/ArrayDepth.java]: can you change this logic

File: mukesh/Arrayoperation.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+	}
+    
+	public static void main(String[] args) {
+	    Scanner sc=new Scanner (System.in);
+	    System.out.println("Enter the length of Array");
+	    int len=sc.nextInt();
+        int []arr=new int[len];
+	    System.out.println("Enter the Array");
+            for(int index1=0;index1<len;index1++)
+            {
```
Comment (pbravichandhar) [inline mukesh/Arrayoperation.java]: indentations

**Pull request merged**